src/libraries/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/ExceptionUtilities.cs

```diff
@@ -14,8 +14,5 @@ internal static Exception UnexpectedValue(object value)
 
             return new InvalidOperationException(SR.Format(SR.UnexpectedValueUnknownType, value));
         }
-
-        internal static Exception Unreachable =>
-            new InvalidOperationException(SR.UnreachableLocation);
```

Comment:
Was this just dead code?  If so, can UnreachableLocation be removed from the resx as well?

Reply:
Yes it can

Reply:
I removed it.